src/sentry/integrations/slack/notifications.py

```diff
@@ -150,6 +150,10 @@ def send_notification_as_slack(
     extra_context_by_actor_id: Mapping[int, Mapping[str, Any]] | None,
 ) -> None:
     """Send an "activity" or "alert rule" notification to a Slack user or team."""
+    if not isinstance(notification, OrganizationNotification):
```

Comment:
@mgaeta wondering why we need this logic if the type of `notification` is `OrganizationNotification`